specification/advisor/resource-manager/Microsoft.Advisor/preview/2016-07-12-preview/advisor.json

```diff
@@ -398,6 +398,72 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/advisorScore": {
```

Comment:
this will be plural. advisorScores.

Can you check with Saket if AdvisorScore resonates with customer scenario. Once this is released, it is very lengthy process to rename. Better to get signoff.